torch/csrc/jit/codegen/cuda/test/test_gpu.cpp

```diff
@@ -25528,6 +25534,81 @@ TEST_F(NVFuserTest, FusionSizeDependentData_CUDA) {
       executor_cache.fusion(), cg_outputs, {a}, {a + 123}, __LINE__, __FILE__);
 }
 
+TEST_F(NVFuserTest, FusionReorderAsRFactor_CUDA) {
```

Comment:
Is the purpose of this test to test `scheduler_utils::domainReorderAsRfactorMap`? If so, should we move it to `test_gpu_scheduler_utils.cpp`?

Reply:
yeah, make sense